captura/test_views.py

```diff
@@ -274,6 +275,100 @@ def test_passing_all_sections(self):
+        capturista.user_set.add(estebes)
+        capturista.save()
+        Escuela.objects.create(nombre='Juan Pablo')
+        self.capturista = Capturista.objects.create(user=estebes)
+        self.browser.visit(self.live_server_url + reverse('tosp_auth:login'))
+        self.browser.fill('username', test_username)
+        self.browser.fill('password', test_password)
+        self.browser.find_by_id('login-submit').click()
+
+    def tearDown(self):
+        """At the end of tests, close the browser.
+        """
+        self.browser.quit()
+
+    def test_creation_of_estudio(self):
+        """ Checks that an estudio can be created through the form
+        in the capturista dashboard
+        """
+        generic_integrante_name = 'Javier'
+        alumno_integrante_name = 'Marco'
+        tutor_integrante_name = 'Fabio'
+        test_url_name = 'captura:estudios'
+        # Test new integrante creation
+        self.browser.visit(self.live_server_url + reverse(test_url_name))
+        self.browser.find_by_id('create_integrante')[0].click()
+        # Test integrante edition
+        self.assertTrue(self.browser.is_text_present('Numero hijos diferentes papas:'))
+        self.browser.fill('numero_hijos_diferentes_papas', 3)
+        self.browser.find_by_id('update_familia')[0].click()
+        self.assertTrue(self.browser.is_text_present('Agregar Integrante'))
+        # Test generic integrante creation
+        self.browser.find_by_id('create_integrante')[0].click()
+        self.assertTrue(self.browser.is_text_present('Edición de Integrante'))
+        self.browser.find_by_id('update_integrante')[0].click()
+        self.assertTrue(self.browser.is_text_present(generic_integrante_name))
+        # Test alumno integrante creation
+        self.browser.find_by_id('create_integrante')[0].click()
+        self.browser.fill('nombres', alumno_integrante_name)
+        self.browser.select('Rol', 'Alumno')
+        self.browser.find_by_id('update_integrante')[0].click()
+        self.assertTrue(self.browser.is_text_present('Numero sae:'))
+        self.browser.find_by_id('update_integrante')[0].click()
+        self.assertTrue(self.browser.is_text_present(alumno_integrante_name))
+        # Test tutor integrante creation
+        self.browser.find_by_id('create_integrante')[0].click()
+        self.browser.fill('nombres', tutor_integrante_name)
+        self.browser.select('Rol', 'Tutor')
+        self.browser.find_by_id('update_integrante')[0].click()
+        self.assertTrue(self.browser.is_text_present('Relacion:'))
+        self.browser.fill('nombres', tutor_integrante_name)
+        self.browser.select('relacion', 'madre')
+        self.browser.fill('nombres', tutor_integrante_name)
+        self.assertTrue(self.browser.is_text_present('Madre'))
+        self.browser.find_by_id('update_integrante')[0].click()
+        self.assertTrue(self.browser.is_text_present(tutor_integrante_name))
+
+    def test_check_invalid_user(self):
```

Comment:
I think these kind of tests can be done w/Django client instead of Splinter. This makes tests faster and accomplish basically the same thing.